common/src/main/java/com/microsoft/identity/common/internal/fido/CredManFidoManager.kt

```diff
@@ -38,15 +45,23 @@ class CredManFidoManager (val context: Context) : IFidoManager {
                                       relyingPartyIdentifier: String,
                                       allowedCredentials: List<String>?,
                                       userVerificationPolicy: String): String {
-        WebAuthnJsonUtil.createJsonAuthRequest(
+        val requestJson = WebAuthnJsonUtil.createJsonAuthRequest(
             challenge,
             relyingPartyIdentifier,
             allowedCredentials,
             userVerificationPolicy
         )
-        // TODO: Implementation will include Credential Manager calls; we aren't able to add the dependency yet,
-        //  so this method will only be filled once we can finish up the passkey work.
-        //  ADO Item: https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2630696
-        return WebAuthnJsonUtil.extractAuthenticatorAssertionResponseJson("")
+        val publicKeyCredentialOption = GetPublicKeyCredentialOption(
```

Comment:
nit: can you add the type information for each variable. I know we can get it by looking at the return type of the method but would be nice if I can see it in place as well.